web/common/transactions.ts

```diff
@@ -23,14 +23,28 @@ export const getAccountTransactions = async (address: string, contractAddress: s
 export const getPendingTransactions = async (address: string, contractAddress: string) => {
   const config = new Configuration({ basePath: network.coreApiUrl });
   const api = new TransactionsApi(config);
-  const txs = await api.getMempoolTransactionList({ limit: 96 });
-  const list = (txs as MempoolTransactionListResponse).results.filter(
-    tx =>
-      tx.tx_type === 'contract_call' &&
-      tx.contract_call.contract_id.split('.')[0] === contractAddress &&
-      tx.sender_address === address &&
-      tx.tx_status === 'pending'
-  );
 
-  return list as MempoolContractCallTransaction[];
+  // Find amount of pages
+  const result = await api.getMempoolTransactionList({ limit: 1 });
+  let pages = Math.ceil(parseFloat(result.total) / 200.0);
+
+  // Go over all pages
+  var swapTransactions = [];
```

Comment:
REQ: use `let` for variable definitions. Use `const` for variables that don't change. I think you can use `const` here.

Reply:
`yarn lint:fix && yarn lint:prettier:fix` and you're good to go.